pkg/routes/logs.go

```diff
@@ -68,3 +72,27 @@ func logFileNameIsTooLong(filePath string) bool {
 	}
 	return false
 }
+
+func prepareLogPath(logPath string) (string, error) {
+	if logPath == ".." ||
+		strings.HasPrefix(logPath, "../") ||
+		strings.HasSuffix(logPath, "/..") ||
+		strings.Contains(logPath, "/../") {
+		return "", fmt.Errorf("invalid path: %q", logPath)
+	}
+	if logPath == "." ||
+		strings.HasPrefix(logPath, "./") ||
+		strings.HasSuffix(logPath, "/.") ||
+		strings.Contains(logPath, "/./") {
+		return "", fmt.Errorf("invalid path: %q", logPath)
+	}
+	if logPath == "" || logPath == "/" {
+		return "", fmt.Errorf("empty path: %q", logPath)
+	}
+	// we ensure that the prefix ends in '/' in below, so skip any leading '/' in the log path now
+	startIndex := 0
+	if logPath[0] == '/' {
+		startIndex = 1
+	}
```

Comment:
Shouldn't we trim all leading slashes? Possibly run `path.Clean()`?

Reply:
+1 on `path.Clean()`. Join, than `path.Clean()`, than check that prefix is still the same